### PR TITLE
includes folder and first include

### PR DESCRIPTION
**Pull request by Rick-Anderson**
[Review URL](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/your-first-mac-aspnet?branch=pr-en-us-3099)

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
DocFX won't take ~ link. I'll see if OPS build does.

**Comment by Rick-Anderson** [comment]
@tdykstra  I'm going to do the all the xplat stuff with includes - can you approve this?

File: aspnetcore/includes/template-files.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (tdykstra) [inline aspnetcore/includes/template-files.md]: shouldn't need metadata, not a separate article

**Comment by Rick-Anderson** [comment]
@tdykstra  I removed **ALL** meta-data

**Pull request merged**